src/govuk/all.js

```diff
@@ -74,6 +81,7 @@ function initAll (options) {
 
 export {
   initAll,
+  I18n,
```

Comment:
This makes sense to me. The reason `I18n` is currently exposed is because it's needed to support localising individually-initialised modules; but in conjunction with your other suggestion, I see no reason for making it private. 

Thanks for the suggestion!